README.md

```diff
@@ -5,190 +5,126 @@ Build Status: [![Build Status](https://travis-ci.org/danmayer/coverband.svg?bran
 
 * Low performance overhead
-* Various controls from sampling, data collection rate, etc to further control performance
-* Ignore directories to avoid overhead data collection on vendor, lib, etc.
+* Very simple setup and configuration
+* Out of the box support for all standard code execution paths (web, cron, background jobs, rake tasks, etc)
+* Easy to understand actionable insights from the report
 * Development mode, offers deep insight of code usage details (number of LOC execution during single request, etc) during development.
-* Easy setup for any Ruby Rack based web framework (Rails, Sinatra, etc)
-* Allows for integration with any other Ruby application flows (background jobs, crons, scripts)
+* Mountable web interface to easily share reports
 
 ## Coverband Demo
 
-Take Coverband for a spin on the live Heroku deployed [Coverband Demo](https://coverband-demo.herokuapp.com/). The [full source code for the demo](https://github.com/danmayer/coverband_demo) is available to help with configuration and understanding of basic usage.
-
-# How To Use
-
-Below is my Coverband workflow, which hopefully will help other best use this library.
-
-* install coverband
-* start your app and hit a few endpoints
-* validate data collection and code coverage with  `rake coverband:coverage`
-* test setup in development (hit endpoints and generate new report)
-* deploy to staging and verify functionality
-* deploy to production and verify functionality
-* every 2 weeks or so, with major releases
-   * clear old coverage: `rake coverband:clear`
-   * deploy and verify coverage is matching expectations
-* __COVERAGE DRIFT__
-   * we are working to address this early in the 3.0.x feature set
-      * https://github.com/danmayer/coverband/issues/118  
-   * if you never clear you have lines of code drift from when they were recorded
-   * if you clear on every deploy you don't capture as useful of data
-   * there is a tradeoff on accuracy and data value
-   * I recommend clearing after major code changes, significant releases, or some regular schedule.
-
-## Example Output
-
-Since Coverband is [Simplecov](https://github.com/colszowka/simplecov) output compatible it should work with any of the `SimpleCov::Formatter`'s available. The output below is produced using the default Simplecov HTML formatter.
-
-Index Page
-![image](https://raw.github.com/danmayer/coverband/master/docs/coverband_index.png)
-
-Details on a example Sinatra app
-![image](https://raw.github.com/danmayer/coverband/master/docs/coverband_details.png)
+Take Coverband for a spin on the live Heroku deployed [Coverband Demo](https://coverband-demo.herokuapp.com/). The [full source code for the demo](https://github.com/danmayer/coverband_demo) is available to help with installation, configuration, and understanding of basic usage.
```

Comment:
I think the demo section should be moved lower. When i look at a readme, i want to see what the gem is and then how to install it.

Reply:
moved down.